examples/dataset/rain_forecast/README.md

```diff
@@ -38,3 +38,31 @@ optional arguments:
   --model {ann,logreg}  Name of model to test.
   --dataset DATASET     Name of dataset to use for testing.
 ```
+
+## Quality Standards Guide
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+test the rain forecast models. See our [QuickStart](https://docs.kolena.io/dataset/quickstart/) guide
+for details.
+
+Here are our [Quality Standards](https://docs.kolena.io/dataset/core-concepts/quality-standard/) recommendations for
+this workflow:
+
+### Metrics
+1. Precision
```

Comment:
Do we support this in QS?

Reply:
We do! Example on [trunk/yoohee](https://trunk.kolena.io/yoohee/dataset/standards?models=N4IglgJiBcDsBM8A0IDGB7AdgMzAcwFcAnAQwBcwsZRIYBmAXwaA&aggregations=N4Ighg5hBOCmFgC4HtogFwgA5wMYEsBnfZAOxABoQALfCagGzusQ1FvtmgElCAhWIkRcMiaAFdYAXyoMwAI1gMMIAAp4iJclSxhoYALaE2IAGZYVRAPoAxVZRCILmawBV7MkIWTjouWCpwhOIMrFJAA&aggregations=N4Ighg5hBOCmFgC4HtogFwjgYzAGzxABoQALASwlL0tMQ1AqtmgEkBnAIVkURY0TQArrAC%2BJPGABGsQpgBKsXAWIgADmGhgAtuwYgAZgDsMIcuwD6AMQByqxGtPmLAFQAKIcSHbIh0bLCmcOxCePSiQA&aggregations=N4Ighg5hBOCmFgC4HtogFwgGYEYD6AzgMaqwgA0IAFgJYRUA2dViGot9s0AkgQEKxEiLhkTQArrAC%2BlBmABGsBhhAAxHAAIAyiTgUQABzDQwAWwJtsAOxU0CeVQDl9WA7fuqACvsRvMdvAAVbxkQAmRxaCIyTDgCcQZWKSA&aggregations=N4Ighg5hBOCmFgC4HtogFzgMZYK7TCwE8QAaEACwEsIKAbGixDUa22aASQGcAhWRIg4ZE0XLAC%2B5OmABGsOhhABBHPkIlyABzAEAttxYgAZgDslVbgH0AYgDkyJrRes2ACo8TnMlqwBUHckRnH2s-DykQbmR8LFglOG5cOmYJIA&aggregations=N4Ighg5hBOCmFgC4HtogFwgLazAOxABoQAbMAI1hI21zwAo4BnAVxMQDoBhAC3wGNYAeQBmAJTABLPAEoiIAA5hoYLEwygA1rACeNXgOHipBAL7EmyFtEE1mbRCFNA&datasetId=15&qualityStandardId=10)